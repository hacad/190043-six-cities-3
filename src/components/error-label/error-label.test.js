import React from "react";
import renderer from "react-test-renderer";
import ErrorLabel from "./error-label.jsx";

it(`ErrorLabel correctly renders after relaunch`, () => {
  const errorLabel = renderer.create(
      <ErrorLabel
        txtLbl="some error text"
        htmlFor="id"
      />).toJSON();

  expect(errorLabel).toMatchSnapshot();
});
