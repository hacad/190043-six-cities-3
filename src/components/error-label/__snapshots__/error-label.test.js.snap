// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorLabel correctly renders after relaunch 1`] = `
<label
  htmlFor="id"
  style={
    Object {
      "color": "red",
    }
  }
>
  some error text
</label>
`;
